src/main/java/mate/academy/bookshop/dto/user/UserRegistrationRequestDto.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.bookshop.dto.user;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import lombok.Data;
+import mate.academy.bookshop.validation.PasswordMatch;
+
+@Data
+@PasswordMatch
+public class UserRegistrationRequestDto {
+    @NotNull
+    @Size(max = 320)
+    @Email
+    private String email;
+    @NotNull
+    @Size(min = 8, max = 30)
+    private String password;
+    @NotNull
+    @Size(min = 8, max = 30)
+    private String repeatPassword;
+    @NotNull
+    @Size(min = 1, max = 30)
+    private String firstName;
+    @NotNull
+    @Size(min = 1, max = 30)
+    private String lastName;

```

Comment:
I wouldn't limit names length, because that is not the thing that user can change to fit our requirements

Reply:
I think we need a limit anyway, I set it to 100, that's enough